x/bank/msgs.go

```diff
@@ -1,4 +1,4 @@
-package simulation
+package bank
```

Comment:
Can we keep this structure as-is/unchanged (i.e. `x/bank/simulation/*`)? I believe this is the pattern we want to follow in modules.

Reply:
I thought we said we wanted to keep only `client` as top-level package?

Reply:
Simulation messages live top-level already, isn't that OK?